js/app.js

```diff
@@ -266,10 +269,10 @@ function App () {
 						modules.push(module.module);
 					}
 				} else {
-					Log.warn("Invalid module position found for this configuration:", module);
+					Log.warn("Invalid module position found for this configuration:" + `\n${JSON.stringify(module, null, 2)}`);
```

Comment:
because it formats the output of the module with the error as a complete block by itself instead of having the open brace on thr end of the line, and the rest on separate lines.